force-app/main/default/objects/npe03__Recurring_Donations_Settings__c/fields/StatusAutomationClosedStatus__c.field-meta.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
@sam-knox Here are the 4 new Status Automation Custom Setting fields which need to be reviewed.